src/content/en/2022/jamstack.md

```diff
@@ -4,10 +4,12 @@ title: Jamstack
 #TODO - Review and update chapter description
 description: Jamstack chapter of the 2022 Web Almanac covers adoption of technology, performance of websites built with Jamstack and weights for various resources.
 authors: [seldo, whitep4nth3r]
-reviewers: [GregBrimble, Nutlope]
-analysts: [seldo, GregBrimble]
-editors: []
+reviewers: [GregBrimble, Nutlope, tunetheweb]
+analysts: [seldo, tunetheweb]
+editors: [DesignrKnight]
 translators: []
+seldo_bio: Laurie has been a web developer since 1996, with occasional breaks to found companies like <a hreflang="en" href="https://www.crunchbase.com/organization/snowball-factory">awe.sm</a> (2010) and <a hreflanh="en" href="https://seldo.com/about/#:~:text=been%20a%20web,learn%20so%20much.">npm</a> (2014). He likes making websites, and he likes making the web bigger and better. He thinks one of the best ways to do that is to encourage more people to do web development, by teaching them existing techniques and by building tools and services that make web development easier, so they don't have to learn so much.
+whitep4nth3r_bio: Salma helps developers build stuff, learn things and love what they do. She works at Netlify as a Staff Developer Experience Engineer, streams live coding, and loves helping people get into tech. After a career as a music teacher and comedian, Salma transitioned to technology in 2014, specializing in front end development and tech leadership for startups, agencies and global e-commerce. <a hreflang="en" href="https://twitch.tv/whitep4nth3r">Find Salma on Twitch</a> to see what she's currently building.
 results: https://docs.google.com/spreadsheets/d/1yfNaj25ToezMwQLKdYP6Qh7AUoX9zMdKMSRVC8JlZMY/
 featured_quote: TODO
 featured_stat_1: TODO
```

Comment:
@seldo @whitep4nth3r we need one Featured Quote, and 3 stats to highlight on the home page (at random, along with the other chapters).

Here's last year's ones:

```
featured_quote: Jamstack young and fast growing technology to build modern websites. That what people say and that's proved by numbers.
featured_stat_1: 1.1%
featured_stat_label_1: Jamstack websites adoption among web
featured_stat_2: 9.5%
featured_stat_label_2: Median value of jQuery usage among Jamstack sites
featured_stat_3: 1.7MB
featured_stat_label_3: Median value of resource wights among Jamstack sites
```

Which looked [like this on the home page](https://almanac.httparchive.org/en/2021/?feat=jamstack#featured-chapter):

<img width="655" alt="image" src="https://user-images.githubusercontent.com/10931297/195334113-f9012268-163a-4bc4-b826-4903825e2e94.png">

Reply:
Some suggestions for the featured quote:

> Although we can’t claim to know exactly what percentage of the web is Jamstack, we can say that around 3% of the web is Jamstack-y, and that this group has been growing strongly for the last 3 years, a great sign for the Jamstack community.

> We believe you can achieve Jamstack-y results with any tech stack, and we hope you find this data useful in identifying tools and platforms that might make it easier to achieve.

Some suggestions for the stats:

```
featured_stat_1: ~3%
featured_stat_label_1: Jamstack-y sites on the web in 2022
featured_stat_2: >100%
featured_stat_label_2: growth of Jamstack-y sites since 2020
featured_stat_3: 36%
featured_stat_label_3: of Jekyll sites are Jamstack-y  
```

Reply:
SGTM!

Have added the first quote and those stats (with slight tweaks) in https://github.com/HTTPArchive/almanac.httparchive.org/pull/3214/commits/1dc3650d3ab0bd8bcab7f2a9f5cdaaec3b5dd963

It looks like this:

<img width="632" alt="image" src="https://user-images.githubusercontent.com/10931297/195342455-9f7bd652-0164-49f1-9fae-f28183b40121.png">

Reply:
That's great, thanks @tunetheweb!

Reply:
Actually changed the 100% to 106% (which is the mobile number) as feels more "real" when not rounded:

<img width="644" alt="image" src="https://user-images.githubusercontent.com/10931297/195343839-cd9a5f33-48e9-460f-9329-4c45f046a0e6.png">

Reply:
Great!